Update 10-batch- requests -.md (#709)
grafana/k6-docs: src/data/markdown/docs/02 javascript api/08 k6-http/10-batch- requests -.md

---

Batch multiple HTTP requests together, to issue them in parallel over multiple TCP connections.

> Note that the batch size can be set as an option [HTTP URL](/using-k6/k6-options/reference/#batch-per-host).

Reviewer: ```suggestion
Note that you can set batch size with the [batch-per-host](/using-k6/k6-options/reference/#batch-per-host) option.
```
